spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/CaffeLoader.scala

```diff
@@ -58,18 +60,41 @@ class CaffeLoader[T: ClassTag](prototxtPath: String, modelPath: String,
     }
   }
 
+  private def loadFromHdfs(fileName: String): FSDataInputStream = {
```

Comment:
rename to another meaningful method name ?  this method just create the inputstream